hr_applicant/models/hr_app_rel_edu.py

```diff
@@ -137,11 +136,11 @@ def create(self, vals):
 
     @api.onchange("from_date", "to_date")
     def _onchange_date(self):
-        to_date = self.to_date and fields.Date.to_string(self.to_date)
-        if (
-            to_date
-            and fields.Date.from_string(to_date) >= fields.Date.today()
-        ):
+
+        to_date = self.to_date 
+        if (to_date and self.to_date >= fields.Date.today()):
```

Comment:
if (self.to_date and self.to_date >= fields.Date.today()):

Reply:
changed to self.to_date